addons/crm_mail_client_extension/views/crm_lead_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+    <!-- Called by the plugin to open a lead in edit mode -->
+        <record id="action_lead_edit_mode" model="ir.actions.act_window">
```

Comment:
Same remark as helpdesk: proçbably to rename to something like "crm_lead_action_form_edit"

Reply:
Done, renamed to "crm_lead_action_form_edit"

@std-odoo, this would require some changes in the gmail plugin code.